bin/oneoff/backfill_data/school_info_id

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env ruby
+
+# This script copies user_id-school_info_id pairs from the pd_enrollments
+# table into the school_info_id column in users.
+
+require_relative '../../../dashboard/config/environment'
+
+# build hash mapping school_info_id to school_id
+school_info_id_school_id_mapping = {}
+
+puts "PROCESSING SCHOOL_INFO"
+SchoolInfo.find_each(batch_size: 2500) do |school_info|
+  school_info_id_school_id_mapping[school_info.id] = school_info.school_id
+  print "."
+end
+puts "\nPROCESSED SCHOOL_INFO"
+
+batch_number = 0
+User.find_in_batches(batch_size: 2_500) do |batch|
```

Comment:
Should this be `User.with_deleted`?

Reply:
Similarly, should the `Pd::Enrollment` below include deleted enrollments?

Reply:
Good questions. @tanyaparker do we care about deleted users and enrollments here? I suppose it can't hurt to include them anyway.

Reply:
Agreed. Let's include them.